moveit_ros/planning_interface/common_planning_interface_objects/src/common_objects.cpp

```diff
@@ -109,6 +110,24 @@ std::shared_ptr<tf2_ros::Buffer> getSharedTF()
   return buffer;
 }
 
+robot_model_loader::RobotModelLoaderPtr getSharedRobotModelLoader(const rclcpp::Node::SharedPtr& node,
+                                                                  const std::string& robot_description)
+{
+  SharedStorage& s = getSharedStorage();
+  boost::mutex::scoped_lock slock(s.lock_);
+  auto it = s.robot_model_loaders_
+                .insert(std::make_pair(node->get_fully_qualified_name() + robot_description,
+                                       robot_model_loader::RobotModelLoaderWeakPtr()))
+                .first;
+  auto rml = it->second.lock();
+  if (!rml)
+  {
+    rml = std::make_shared<RobotModelLoader>(node, robot_description);
```

Comment:
I assume you removed `opt.load_kinematics_solvers_ = true` on purpose since we don't call `setFromIK()` anywhere? But I think that RobotInteraction depends on IK solvers being loaded.

Reply:
Sorry for not clarifying it, I did remove it because loading the kinematics solvers is the default for robot model loader https://github.com/ros-planning/moveit2/blob/main/moveit_ros/planning/robot_model_loader/include/moveit/robot_model_loader/robot_model_loader.h#L56, if you want me to add it back I can do it immediately

Reply:
Oh, I wasn't aware of them being enabled by default. All good, thanks!